src/rt/ds/hashmap.h

```diff
@@ -355,7 +355,9 @@ namespace verona::rt
 
       // Maximum probe length reached, resize and retry.
       resize(alloc);
-      return insert(alloc, std::forward<E>(entry));
+      // Entry may have been swapped prior to resize.
+      insert(alloc, std::forward<E>(entry));
+      return std::make_pair(true, find((const KeyType*)key));
```

Comment:
This will incur in two map searches, right? Perhaps check the result of `insert` and only do that if it actually changes?

Reply:
Fixed